src/centos/scripts/start-marklogic.sh

```diff
@@ -28,13 +28,23 @@ for var in OVERWRITE_ML_CONF INSTALL_CONVERTERS MARKLOGIC_DEV_BUILD MARKLOGIC_IN
     declare $var="$(echo "${!var}" | sed -e 's/[[:blank:]]//g' | awk '{print tolower($0)}')"
 done
 
-# define log and err functions
-log() {
-    echo "$(basename "${0}"): ${*}"
+#log utility
+info() {
+    log "Info" "$@"
 }
-err() {
-    echo "$(basename "${0}") ERROR: ${*}" >&2
+error() {
+    log "Error" "$1"
```

Comment:
Could we change the message back to "ERROR: [message]" instead of "Error[message]"? I think the first option is better for readability when debugging

Reply:
That's the sentence case followed in MarkLogic server logs as well, hence I kept the format for logs the same.
Following line is the log from the server: 
2022-06-23 21:16:53.859 **Info**: Forest Schemas state changed from recovering to open